pkgs/applications/science/electronics/caneda/default.nix

```
@@ -12,7 +12,7 @@ mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake ];
-  buildInputs = [ qtbase qttools qtsvg qwt ];
+  buildInputs = [ qtbase qttools qtsvg libsForQt5.qwt6_1 ];
```

```suggestion
  buildInputs = [ qtbase qttools qtsvg qwt6_1 ];
```
This package is using libsForQt5 callPackage. We should not input another libsForQt5.